context/app/static/js/components/files/GlobusLink/GlobusLink.jsx

```diff
@@ -24,7 +24,12 @@ function GlobusLink(props) {
           headers: requestHeaders,
         });
         if (!response.ok) {
-          console.error('Entities API failed', response);
+          if (response.status === 403) {
```

Comment:
Should we check for existence of a nexus token?

Reply:
I don't know that that would change anything: You could be logged in, but still not have sufficient privs, and a 403 would still be an expected response. 

If I'm misunderstanding, let me know?